otter/test/rest/test_groups.py

```diff
@@ -860,7 +899,7 @@ def test_view_manifest_with_webhooks(self):
         for i in [0, 1]:
             exp_policies[i]['webhooks'] = deepcopy(webhooks[i])
             for j, webhook in enumerate(exp_policies[i]['webhooks']):
-                exp_policies[i]['webhooks'][j]['links'] = webhooks_internal_links[i][j]
```

Comment:
Github is still showing this comment because it's attached to the deleted line, and the line is still deleted. However, the problem (name change) has been addressed.